src/config/config_boston.yml

```diff
@@ -39,8 +39,11 @@ crashes_files:
         optional:
             summary: location_type
             address: 
-            vehicles: mode_type
-            bikes: mode_type
+            mode:
```

Comment:
Why do none of these include MV? It seems like with the current method there would be nothing populated for MV?

Reply:
I also generally worry here: Why are we building the structure specifically to address these three modes? It seems like it might make more sense to set this up as a "split" feature or a "crash" feature and be agnostic to what the different values of that mode column might be.